### PR TITLE
Add ABI dump script

### PR DESCRIPTION
**Pull request by pablogsal**

### CODE REVIEW

File: .github/workflows/regen-abidump.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+set -e
```
Comment (encukou) [inline .github/workflows/regen-abidump.sh]: Let's make it print out what it's doing?
```suggestion
set -ex
```

File: .github/workflows/regen-abidump.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+./.github/workflows/posix-deps-apt.sh
```
Comment (encukou) [inline .github/workflows/regen-abidump.sh:4]: This asks me where I live when it installs tzdata. That shouldn't be necessary.
```suggestion
export DEBIAN_FRONTEND=noninteractive
./.github/workflows/posix-deps-apt.sh
```

**Review by encukou — APPROVED** [review]
Looks good, two little improvements below.

I checked with `podman run -v$(pwd):/src:Z -w /src --rm -it ubuntu:20.04 bash /src/.github/workflows/regen-abidump.sh`. Two changes to carry over to the Docker command line:
- ` -w /src` for the working directory
- ` --rm` to remove the container:
  - If you aren't familiar with containers you probably don't want used ones lying around.
  - If you do want to keep it around, you probably know what `--rm` is and can remove it.

**Comment by pablogsal** [comment]
Thanks a lot for the review @encukou

**Comment by pablogsal** [comment]
I will make the PR to fix the devguide soon

**Pull request merged**